spring-integration-mongodb/src/test/java/org/springframework/integration/mongodb/store/ConfigurableMongoDbMessageStore-CustomConverter.xml

```diff
@@ -29,6 +29,7 @@
 
 	<beans:bean id="messageStore" parent="abstractMessageStore">
 		<beans:constructor-arg name="mappingMongoConverter" ref="customConverter"/>
+		<beans:property name="createIndex" value="false" />
```

Comment:
When I add the property, the createIndex value is still true after the application context creation. However, adding to the `abstractMessageStore` set the value to corresponding value in this case to false.

Reply:
That’s because a super class creates its own indexes and we don’t check for this property there. See our discussion in that class